en/streaming-integrator/docs/ref/performance-analysis-results.md

```diff
@@ -3,6 +3,33 @@
 !!!note
     These performance statistics were taken when the load average was below 3.8 in the 4 core instance.
 
+    
+## Performance analysis results summary
+
+The recommended CPU and memory specifications for Docker containers are as follows:
+
+- **CPU**: 4 cores
+
+- **Memory**: 8GB
+
+- **Xms**: 2GB
+
+- **Xmx**: 4GB
```

Comment:
This content has to come under 
"The recommended memory specifications SI server as follows and should be configured in {SI_Home}/wso2/server/bin/carbon.sh

Reply:
Added the mentioned text.